app/views/issues/new.html.slim

```diff
@@ -6,15 +6,15 @@ h1 Create new issue
       = simple_form_for @issue, html: {multipart: true} ,
                                 url: issues_path do |f|
         = f.error_messages
-        = f.input :name, label: 'Name'
+        = f.input :name, label: 'Name',
                     placeholder: 'Vasya Pupkin'
-        = f.input :address, label: 'Address'
+        = f.input :address, label: 'Address',
                     placeholder: 'Khotkevycha str. 33/3'
-        = f.input :phone, label: 'Phone'
+        = f.input :phone, label: 'Phone',
                     placeholder: '+380 342 77 77 77'
         = f.input :email, as: :email, label: 'Email',
                     placeholder: 'user@domain.com'
-        = f.input :category, label: 'Category'
+        = f.input :category_id, collection: @categories, label: 'Category'
```

Comment:
change to f.association :category, collection: Category.ordered.all

where `ordered` is a new scope (order by name from a to ya)